src/TodoListItem.jsx

```diff
@@ -1,6 +1,9 @@
-function TodoListItem(props){
+function TodoListItem({id, title,onRemoveTodo}){
         return(
-                <li>{props.todo.title}</li>              
+                <li>               
+                        {title}
```

Comment:
Actually I suggest installing prettier  as there are inconsistencies in the gaps (or indentations) in the components. It will help you a lot with formatting.
<img width="606" alt="Screenshot 2024-07-30 at 18 40 04" src="https://github.com/user-attachments/assets/ea1d099a-f0e8-490d-bf41-750013c6746e">
<img width="700" alt="Screenshot 2024-07-30 at 18 40 11" src="https://github.com/user-attachments/assets/ff510345-0b46-484d-8d5e-fdc75ffc1e44">
<img width="693" alt="Screenshot 2024-07-30 at 18 40 19" src="https://github.com/user-attachments/assets/70481a0b-d6a5-4eda-adb4-6002763b0592">